src/lib/createables.ts

```diff
@@ -676,6 +679,20 @@ const Createables: Createable[] = [
 			"Ava's assembler": 1
 		}),
 		QPRequired: 205
+	},
+	{
+		name: 'Dragon sq shield',
+		inputItems: resolveNameBank({
+			'Shield right half': 1,
+			'Shield left half': 1
+		}),
+		outputItems: resolveNameBank({
+			'Dragon sq shield': 1
+		}),
+		QPRequired: 111,
+		smithingLevel: 60,
+		xpGain: 75,
+		xpSkill: SkillsEnum.Smithing
```

Comment:
this `xpGain` thing isnt coded the best, what if it gives XP in more than 1 thing? i think just remove the xp reward honestly because he xp reward is so so minimal that its hardly worth even coding it, personally. if you really want to though, it needs to use a *typed* object like this: `{ smithing: 60 }`